packages/block-editor/src/store/selectors.js

```diff
@@ -2089,7 +2119,7 @@ export const getInserterItems = createSelector(
 		state.blockListSettings[ rootClientId ],
 		state.blocks.byClientId,
 		state.blocks.order,
-		state.preferences.insertUsage,
+		__unstableGetInsertUsage(),
```

Comment:
Double checking: are the objects returned by `get` stable? Referential equality is used by `rememo` to determine if cache should be invalidated.

Reply:
The values are from the preferences store, and those values only change when the `insertUsage` preference is updated, so should be no issues.